src/sync/installation.ts

```diff
@@ -458,14 +458,11 @@ export const processInstallation =
 
 			const nextTaskDelays: Array<number> = [];
 
-			const includeHostnameToDedupKey = await booleanFlag(BooleanFlags.INCLUDE_JIRA_HOSTNAME_TO_DEDUP_KEY, false, jiraHost);
-
-			const result = await deduplicator.executeWithDeduplication(includeHostnameToDedupKey
-				? "i-" + installationId
-				: "i-" + installationId + "-" + jiraHost,
-			() => doProcessInstallation(app, job, installationId, jiraHost, logger, (delay: number) =>
-				nextTaskDelays.push(delay)
-			));
+			const result = await deduplicator.executeWithDeduplication(
+				"i-" + installationId + "-" + jiraHost,
```

Comment:
Shouldn't this be the other branch of the ternary above? Or was the logic in the ternary the wrong way around?